Source/DafnyStandardLibraries/scripts/check-examples

```diff
@@ -63,6 +63,11 @@
 ##              "dafny run --use-basename-for-filename $F" and,
 ##              if <expect> is present, checks that the output matches the
 ##              content of the file names <expect>
+##   %check-run-output <expect> : Like %check-run, runs
+##              "dafny run --use-basename-for-filename $F" and,
+##              if there is an output, it will ignore it
+##   %check-expect : compares the output against the output of the previous
+##              %check-run-output command
```

Comment:
Was just this for your local debugging or is it used in CI now? I don't love the statefulness if the latter.

Reply:
It is use in the CI of this PR. In the README, it makes it possible to present a program and say "here is this program's output" and have that program's output be tested against the provided program. Otherwise outputs are not tested. In a README, we usually use two code blocks for that. I don't know how to do it in a non-stateful way since the loop takes every block separately.